src/dataServices/dynamoDbDataService.ts

```diff
@@ -35,7 +35,25 @@ import { DynamoDbBundleService } from './dynamoDbBundleService';
 import { DynamoDbUtil } from './dynamoDbUtil';
 import DynamoDbParamBuilder from './dynamoDbParamBuilder';
 import DynamoDbHelper from './dynamoDbHelper';
-import { getBulkExportResults } from '../bulkExport/bulkExportResults';
+import { getBulkExportResults, startJobExecution } from '../bulkExport/bulkExport';
+import { BulkExportJob } from '../bulkExport/types';
+
+const buildExportJob = (initiateExportRequest: InitiateExportRequest): BulkExportJob => {
+    const initialStatus: ExportJobStatus = 'in-progress';
+    return {
+        jobId: uuidv4(),
+        jobOwnerId: initiateExportRequest.requesterUserId,
+        exportType: initiateExportRequest.exportType,
+        groupId: initiateExportRequest.groupId ?? '',

```

Comment:
No changes required -- interesting usage I haven't heard of [nullish-coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing)

Any concerns with this being our default on how to default set a parameter?

Reply:
yes, using it for assigning default values is maybe it's most popular use case. There aren't any downsides in using it.